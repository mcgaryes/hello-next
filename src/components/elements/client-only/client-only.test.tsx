import {render} from '@testing-library/react'
import ClientOnly from "@/elements/client-only/client-only";

test('Renders client-only unchanged', () => {
    const {container} = render(<ClientOnly><div/></ClientOnly>)
    expect(container).toMatchSnapshot()
})
