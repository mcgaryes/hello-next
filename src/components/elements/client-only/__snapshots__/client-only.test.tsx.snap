// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders client-only unchanged 1`] = `
<div>
  <div>
    <div />
  </div>
</div>
`;
